test/config/MockAppConfig.scala

```diff
@@ -84,6 +84,8 @@ class MockAppConfig extends AppConfig with MockFactory {
 
   override lazy val giftAidReleased: Boolean = true
 
+  override lazy val nrsEnabled: Boolean = false
```

Comment:
Should this not reflect the state in application config?

Reply:
Changes requested have been added